src/json-types.ts

```diff
@@ -29,6 +30,37 @@ export interface Session {
   user: SessionUser
 }
 
+export interface RequestContext {
+  System: System
+  AudioPlayer: AudioPlayer
+}
+
+export interface System {
+  application: SessionApplication
+  user: SessionUser
+  device: SystemDevice
+  apiEndpoint: string
+}
+
+export interface SystemDevice {
+  deviceId: string
+  supportedInterfaces: SupportedInterfaces
+}
+
+export interface SupportedInterfaces {
+  [key: string]: object
+}
+
+export interface SessionApplication {
+  applicationId: string
+}
+
+export interface AudioPlayer {
+  token: string
+  offsetInMilliseconds: number
```

Comment:
> This is only included in the AudioPlayer object when your skill was the skill most recently playing audio on the device. 

Sorry, I'm not able to run this myself right now to check it - could you confirm if this means the `offsetInMilliseconds` and `token` are optional parameters - will they be undefined if your skill was not the most recent to start playing audio?